src/Components/SuperAdmins/Form/index.js

```diff
@@ -165,19 +99,20 @@ const FormSuperAdmin = () => {
       });
       setAddVisible(true);
       setSaveVisible(false);
+      setIsUserCreated(true);
     }
   };
 
   const closeForm = () => {
-    if (isUserCreated) {
+    if (!isUserCreated) {
```

Comment:
that was probably fine in this way
```suggestion
    if (isUserCreated) {
```

Reply:
If i delete the (!) , when we create the user, dont redirect to the home! Its necessary!